demos/python_demos/llm_text_generation/servable_stream/config.py

```diff
@@ -71,114 +65,265 @@ def youri_partial_text_processor(partial_text, new_text):
     return partial_text
 
 
+def internlm_partial_text_processor(partial_text, new_text):
+    partial_text += new_text
+    return partial_text.split("<|im_end|>")[0]
+
+
 SUPPORTED_LLM_MODELS = {
```

Comment:
Can you add a comment about the commented parts if we don't want to remove them?